deploy/ansible/roles/iroha-cluster-config-gen/templates/peers.list.j2

```diff
@@ -0,0 +1,5 @@
+{% for host in groups['all'] | difference([inventory_hostname]) | sort(reverse=True) %}
```

Comment:
If possible, can you explain with comments how iroha/deploy/ansible/roles/iroha-cluster-config-gen/templates/peers.list.j2 template is formed?

Reply:
@neewy sure, will be added in next commit